core/controllers/ingest.py

```diff
@@ -199,6 +208,80 @@ def populate_db(workbook: dict[str, pd.DataFrame], mappings: tuple[DataMapping])
     db.session.commit()
 
 
+def populate_db_round_one(workbook: dict[str, pd.DataFrame], mappings: tuple[DataMapping]) -> None:
```

Comment:
Should we give this a name to indicate it's specific to Towns Fund (as other funds may have a Round One in future?